perun-core/src/main/java/cz/metacentrum/perun/core/bl/PerunBl.java

```diff
@@ -256,4 +256,10 @@
 	 * @return ConfigManagerBl
 	 */
 	ConfigManagerBl getConfigManagerBl();
+
+	/**
+	 * Gets a ConsentsManagerBl
+	 * @return ConsentsManagerBl
+	 */
+	ConsentsManagerBl getConsentsManagerBl();
```

Comment:
I think getConsentsManager() method should be here too, same as other managers have it.

Reply:
Done, I added missing getConfigManager() as well.